ports/rp2/boards/PI-PLATES_MICROPI/board.json

```
@@ -0,0 +1,25 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Pi-Plate Compatible",
+        "Data Acquisition",
+        "Micro USB",
+        "Real Time Clock",
+        "OLED Display",
+        "Temperature Sensor",
+        "LED Annunciators",
+        "microSD Storage"
+    ],
```

It's encouraged to use the subset of features available in the [MicroPython downloads](https://micropython.org/download/) page.

At a first glance, the following looks appropriate:

```suggestion
    "features": [
        "USB",
        "Display",
        "Environment Sensor",
        "microSD"
    ],
```